pdc/apps/component/serializers.py

```diff
@@ -373,32 +373,6 @@ def to_internal_value(self, data):
             data['global_component'] = self.instance.global_component
         return super(ReleaseComponentSerializer, self).to_internal_value(data)
 
-    def validate_release(self, value):
-        if not isinstance(value, Release):
-            if isinstance(value, dict):
-                release_id = value['release_id']
-            else:
-                release_id = value
-            if release_id is None or release_id.strip() == "":
-                self._errors = {'release': 'This field is required.'}
-                return
-            release = get_object_or_404(Release, release_id=release_id)
-            if not release.is_active():
```

Comment:
If we need to keep the check of "Not create release component with an inactive release", we'd better keep `validate_release` here.
The reason that test [here](https://github.com/release-engineering/product-definition-center/blob/master/pdc/apps/component/tests.py#L1747-L1755) get passed is because the 'releases' in data is should be 'release'.
Please correct other tests of `ReleaseComponent` that with 'releases' in data as well.